lang/java/BUILD.md

```diff
@@ -36,6 +36,8 @@ The Java side of REEF is built using Apache Maven. To build and run tests, execu
 
     mvn clean install
 
+On an ubuntu vm running version 16.10 the build typically takes 25-30 minutes if you include all tests
```

Comment:
Maven runs builds in parallel so the build time is highly dependent on the number cores available so you should also include the number of cores allotted to the the VM.